pkg/authorization/apiserver/apiserver.go

```diff
@@ -142,3 +156,22 @@ func (c *completedConfig) newV1RESTStorage() (map[string]rest.Storage, error) {
 	v1Storage["roleBindingRestrictions"] = roleBindingRestrictionStorage
 	return v1Storage, nil
 }
+
+func (c *completedConfig) v1alpha1RESTStorage() (map[string]rest.Storage, error) {
```

Comment:
This construct isn't needed since you're only called once.  `v1` needs it because legacy storage re-uses v1 storage